lib/tasks/migration.rake

```diff
@@ -241,20 +252,20 @@ namespace :migration do
       date_submitted ||= dc_version.xpath("dcterms:datesubmitted", NS).text() if dc_version.xpath("dcterms:datesubmitted", NS)
       is_version_of = dc_version.xpath("dcterms:isVersionOf", NS).text() unless dc_version.xpath("dcterms:isVersionOf", NS).blank?
       is_version_of ||= dc_version.xpath("dcterms:isversionof", NS).text() if dc_version.xpath("dcterms:isversionof", NS)
-      graduation_date = dc_version.xpath("ualterms:graduationdate", NS).text() if dc_version.xpath("ualterms:graduationdate", NS)
-      specialization = dc_version.xpath("ualterms:specialization", NS).text() if dc_version.xpath("ualterms:specialization", NS)
+      graduation_date = dc_version.xpath("ualdate:graduationdate", NS).text() if dc_version.xpath("ualdate:graduationdate", NS)
+      specialization = dc_version.xpath("ualthesis:specialization", NS).text() if dc_version.xpath("ualthesis:specialization", NS)
       supervisors = dc_version.xpath("marcrel:ths/text()", NS).map(&:text) if dc_version.xpath("marcrel:ths", NS)
-      committee_members = dc_version.xpath("ualterms:thesiscommitteemember/text()", NS).map(&:text) if dc_version.xpath("ualterms:thesiscommitteemember/text()", NS)
+      committee_members = dc_version.xpath("ualrole:thesiscommitteemember/text()", NS).map(&:text) if dc_version.xpath("ualrole:thesiscommitteemember/text()", NS)
       departments = dc_version.xpath("vivo:AcademicDepartment/text()", NS).map(&:text) if dc_version.xpath("vivo:AcademicDepartment", NS)
       thesis_name = dc_version.xpath("bibo:ThesisDegree", NS).text() if dc_version.xpath("bibo:ThesisDegree", NS)
-      thesis_level = dc_version.xpath("ualterms:thesislevel", NS).text() if dc_version.xpath("ualterms:thesislevel", NS)
+      thesis_level = dc_version.xpath("ualthesis:thesislevel", NS).text() if dc_version.xpath("ualthesis:thesislevel", NS)
       alternative_titles = dc_version.xpath("dcterms:alternative/text()", NS).map(&:text) if dc_version.xpath("dcterms:alternative", NS)
-      proquest = dc_version.xpath("ualterms:proquest", NS).text() if dc_version.xpath("ualterms:proquest", NS)
-      unicorn = dc_version.xpath("ualterms:unicorn", NS).text() if dc_version.xpath("ualterms:unicorn", NS)
+      proquest = dc_version.xpath("ualid:proquest", NS).text() if dc_version.xpath("ualid:proquest", NS)
+      unicorn = dc_version.xpath("ualid:unicorn", NS).text() if dc_version.xpath("ualid:unicorn", NS)
       degree_grantor = dc_version.xpath("marcrel:dgg", NS).text() if dc_version.xpath("marcrel:dgg", NS)
       dissertant = dc_version.xpath("marcrel:dis", NS).text() if dc_version.xpath("marcrel:dis", NS)
       dissertant = creators.first if type == "Thesis" && (dissertant.nil? || dissertant.blank?)
```

Comment:
The check of type is to avoid date_accepted and date_submitted to be migrated for non-thesis objects, as they were bugs in old ERA and some regular ERA objects have that data as well. But the case should be "Thesis", pushing a change to the PR right now. @pbinkley